src/AtelierTomato.Markov.Storage.Sqlite/SqliteSentenceAccess.cs

```diff
@@ -134,7 +141,7 @@ public async Task WriteSentenceRange(IEnumerable<Sentence> sentenceRange)
 
 		private async Task WriteCore(Sentence sentence, SqliteConnection connection)
 		{
-			SentenceRaw sentenceRaw = new(sentence);
+			SentenceRow sentenceRaw = new(sentence);

```

Comment:
variable name should also be ..row if you renamed the type, since that's what it is